storm-core/src/jvm/org/apache/storm/Config.java

```diff
@@ -140,6 +140,45 @@
     public static final String STORM_META_SERIALIZATION_DELEGATE = "storm.meta.serialization.delegate";
 
     /**
+     * A list of daemon metrics  reporter plugin class names. The classes should implement
```

Comment:
incomplete sentence for config explanation. Guess it is not required "The classes should implement"